### DISCUSSION TITLE
Dust emission factor and namelist changes

### DISCUSSION THREAD

**Discussion by tilmes** [NCAR/CARMA_dev#9] [trop_strat CARMA model]
A new dust emission factor namelist variable has been implemented for CARMA WACCM-MA 2deg model. This needs to be also implemented for CAMchem. The namelist changes are consistent with what has been used in the CARMA runs in the first development paper. This and discussion #4 would go in a new tag, to start running issue #1 .

@fvitt could you implement the following based on this case and also make a new 
/glade/p/cesm/chwg_dev/tilmes/cases/carma/FWmaCARMAnudged.f19_f19_mg17.carma_trop_strat06.1990_2000.test_dusttuning
1. new namelist changes as in user_nl_cam for both  WACCM-MA 2deg nudged and F compsets
2. dust emission factor of 0.5e+09 for WACCM-MA 1deg version. Other namelist  changes are still tested (will be added to this discussion in this discussion (tbd if more changes go in before making a new tag)
3. add the dust emission factor changes also to the CAMchem CARMA compsets (0.5e+09 factor for CAMchem 1deg)

**Comment by tilmes** [comment]
@fvitt has this been added now to carma_trop_strat09?

**Comment by fvitt** [comment]
@tilmes 
Yes tag carma_trop_strat09 includes these changes and the changes for hydrophobic back carbon described in
https://github.com/NCAR/CARMA_dev/discussions/4

**Reply by tilmes** [reply]
@fvitt  Could you addd some more changes for the FWmaCARMAHIST compset I found:
lght_no_prd_factor             = 1.00D0

srf_emis_specifier             = 'CH2O -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015_2deg/emissions-cmip6_CH2O_anthro_surface_1750-2015_1.9x2.5_c20170608.nc',
         'CH2O -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015_2deg/emissions-cmip6_CH2O_bb_surface_1750-2015_1.9x2.5_c20170322.nc',
         'CO -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015_2deg/emissions-cmip6_CO_anthro_surface_1750-2015_1.9x2.5_c20180504.nc',
         'CO -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015_2deg/emissions-cmip6_CO_bb_surface_1750-2015_1.9x2.5_c20170322.nc',
         'CO -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015_2deg/emissions-cmip6_CO_other_surface_1750-2015_1.9x2.5_c20170322.nc',
         'NO -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015_2deg/emissions-cmip6_NO_anthro_surface_1750-2015_1.9x2.5_c20170608.nc',
         'NO -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015_2deg/emissions-cmip6_NO_bb_surface_1750-2015_1.9x2.5_c20170322.nc',
         'NO -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015_2deg/emissions-cmip6_NO_other_surface_1750-2015_1.9x2.5_c20170322.nc',
         'DMS -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015_2deg/emissions-cmip6_DMS_bb_surface_1750-2015_1.9x2.5_c20170322.nc',
         'DMS -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015_2deg/emissions-cmip6_DMS_other_surface_1750-2015_1.9x2.5_c20170322.nc',
         'SO2 -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015_2deg/emissions-cmip6_SO2_anthro-ag-ship-res_surface_1750-2015_1.9x2.5_c20170616.nc',
         'SO2 -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015_2deg/emissions-cmip6_SO2_anthro-ene_surface_1750-2015_1.9x2.5_c20170616.nc',
         'SO2 -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015_2deg/emissions-cmip6_SO2_bb_surface_1750-2015_1.9x2.5_c20170322.nc',
         'SOAG -> 0.3*/glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015_2deg/emissions-cmip6_SOAGx1.5_anthro_surface_1750-2015_1.9x2.5_c20170608.nc',
         'SOAG -> 0.3*/glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015_2deg/emissions-cmip6_SOAGx1.5_bb_surface_1750-2015_1.9x2.5_c20170322.nc',
         'SOAG -> 0.1*/glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015_2deg/emissions-cmip6_SOAGx1.5_biogenic_surface_1750-2015_1.9x2.5_c20170322.nc'

 ext_frc_specifier              = 'NO2 -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015_2deg/emissions-cmip6_NO2_aircraft_vertical_1750-2015_1.9x2.5_c20170608.nc',
         'SO2 -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015_2deg/emissions-cmip6_SO2_aircraft_vertical_1750-2015_1.9x2.5_c20170608.nc',
         'SO2 -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015_2deg/emissions-cmip6_SO2_contvolcano_vertical_850-5000_1.9x2.5_c20190417.nc'
'SO2 -> /glade/p/acom/acom-climate/cesm2/inputdata/atm/cam/chem/stratvolc/VolcanEESMv3.21_SO2_1979-2022_NoPinatubo1991_2deg_c20220331.nc',
         'SO2 -> /glade/p/acom/acom-climate/cesm2/inputdata/atm/cam/chem/stratvolc/Pinatubo_1991061506-1991061515_19-27kmProfile_5.2S-15.1N_SO2_10Tg_2deg_c20220417.nc'

**Reply by tilmes** [reply]
and also
&ocean_emis_nl
 bubble_mediated_transfer               = .FALSE.
 csw_specifier          = 'DMS -> /glade/work/siyuan/Csw_DMS_Lana2011_f09f09_1750_2100_20200717a.nc'
 csw_time_type          = 'SERIAL'
 ocean_salinity_file            = '/glade/work/siyuan/SSS_recooked_0-360.nc'
/
solar_parms_data_file          = '/glade/p/cesmdata/cseg/inputdata/atm/cam/solar/SolarForcingCMIP6_18491230-23000102_c20200615.nc'

Also, I noticed that these, should be actually removed from all the surface emissions:
'DMS -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015_2deg/emissions-cmip6_DMS_other_surface_1750-2015_1.9x2.5_c20170322.nc',
